### ISSUE TITLE
Circular dependency array.js <=> value.js

### ISSUE THREAD

**Issue by darrickyee** [hybridsjs/hybrids#110]
I am receiving a "circular dependency" warning when bundling hybrids using rollup.js:

```
(!) Circular dependency
node_modules\hybrids\esm\template\resolvers\value.js -> node_modules\hybrids\esm\template\resolvers\array.js -> node_modules\hybrids\esm\template\resolvers\value.js
```

It seems `value.js` imports `resolveArray()` from `array.js` and calls it in `resolveValue()`, while `array.js` imports `resolveValue()` and calls it in `resolveArray()`.

I don't know how much of a problem this is (it hasn't bitten me yet), but thought I'd bring it up as it seems like it could potentially cause problems down the road.

[Edit] Actually, I am getting an error "resolveValue is not a function" here - maybe it's caused by this issue?  Or maybe I've messed something up.  (I believe the circular dependency is in the template engine, which I don't normally use)

https://codesandbox.io/s/crazy-curran-k24p7?file=/src/index.js

**Comment by smalluban** [comment]
I discovered that it is a problem in codesandbox a few days ago. Still, it is not a bug, ES modules support circular dependencies, and if they are used wisely everything should be fine. 

Because of lack of support in codesandbox (and you wrote about rollup) I've changed imports between those files. I'll release the new patch version soon (the commit is already on `master` branch). I'll notify you here then.
